module/Activity/src/Activity/Mapper/Activity.php

```diff
@@ -83,6 +83,10 @@ public function getUpcomingActivities($count = null, $organ = null)
                 ->setParameter('organ', $organ);
         }
 
+        // For now 'career' is the only category, however this may change in the future
+        if ($category === 'career') {
+            $qb->andWhere('a.isMyFuture = 1');
```

Comment:
It would be good to document this additional feature of 'myfuture' in the tooltip in the activity create form. Perhaps it is more consistent to rename some more things to 'career'.